Tasks/AzureMysqlDeployment/task.json

```diff
@@ -0,0 +1,231 @@
+        },
+        {
+            "name": "DatabaseName",
+            "type": "string",
+            "label": "Database Name",
+            "required": false,
+            "groupName": "target",
+            "defaultValue": "",
+            "helpMarkDown": "The name of database on which the script is needed to be run​."
+        },
+        {
+            "name": "SqlUsername",
+            "type": "string",
+            "label": "Server Admin Login",
+            "required": false,
+            "groupName": "target",
+            "defaultValue": "",
+            "helpMarkDown": "Azure Database for MySQL server supports native MySQL authentication.<br>You can connect and authenticate to a server with the server's admin login. Example:  bbo1@fabrikam."
+        },
+        {
+            "name": "SqlPassword",
+            "type": "string",
+            "label": "Password",
+            "required": false,
+            "groupName": "target",
+            "defaultValue": "",
+            "helpMarkDown": "Administrator password for Azure DB for Mysql. In case you don’t recall the password you can change the password from [Azure portal](https://docs.microsoft.com/en-us/azure/mysql/howto-create-manage-server-portal).<br>It can be variable defined in the definition. Example : $(password).<br>Also, you may mark the variable type as 'secret' to secure it."
+        },
+        {
+            "name": "TaskNameSelector",
+            "type": "pickList",
+            "label": "Type",
+            "required": false,
+            "helpMarkDown": "",
+            "groupName": "taskDetails",
+            "defaultValue": "SqlTask",
+            "options": {
+                "SqlTask": "MySQL Script File",
+                "InlineSqlTask": "Inline MySQL Script"
+            }
+        },
+        {
+            "name": "SqlFile",
+            "type": "filePath",
+            "label": "MySQL Script",
+            "required": true,
+            "groupName": "taskDetails",
+            "defaultValue": "",
+            "helpMarkDown": "Full path of the script file on the automation agent or on a UNC path accessible to the automation agent like,  \\\\\\\\BudgetIT\\DeployBuilds\\script.sql. Also, predefined system variables like, $(agent.releaseDirectory) can also be used here. A file containing SQL statements can be used here.​",
+            "visibleRule": "TaskNameSelector = SqlTask"
+        },
+        {
+            "name": "SqlInline",
+            "type": "multiLine",
+            "label": "Inline MySQL Script",
+            "required": true,
+            "groupName": "taskDetails",
+            "defaultValue": "",
+            "helpMarkDown": "Enter the MySQL script to execute on the Database selected above.",
+            "visibleRule": "TaskNameSelector = InlineSqlTask",
+            "properties": {
+                "resizable": "true",
+                "rows": "10"
+            }
+        },
+        {
+            "name": "SqlAdditionalArguments",
```

Comment:
Why there is a duplication here `SqlAdditionalArguments `and `InlineAdditionalArguments`?

Reply:
Same pattern from Sql task as we are getting additional argument for each one separately. Yes we can do with one field also but it involve lots of change so I am not taking it in this release.

Reply:
Done . Now it has one input .